lib/Serializer/functions.php

```diff
@@ -165,8 +164,6 @@ function standardSerializer(Writer $writer, $value)
     } elseif (is_callable($value)) {
         // A callback
         $value($writer);
-    } elseif (is_null($value)) {
```

Comment:
phpstan complains:
```
167    Call to function is_null() with array|object will always evaluate to false
```
It knows that `$value` at this point might be an array or object.

After looking at the code, there are `elseif` cases below that explicitly handle the `array` and `object` cases. So I refactored so that `is_null()` is not needed here.